sql/_23_apricot_qa/_01_sql_extension3/_05_analytic_functions/_04_max/cases/max_group_by.master.slave1.diff_1

```diff
@@ -1,66 +1,66 @@
-1850d1849
+1852d1851
 <   'db_attribute'        'col1'                'max_groupby_order'   'INSTANCE'                      0  NULL                  NULL                  'INTEGER'                      10            0  'Not applicable'      'Not applicable'      NULL                  NULL                  'NO'                  NULL                
-1852d1850
+1854d1852
 <   'db_attribute'        'col2'                'max_groupby_order'   'INSTANCE'                      1  NULL                  NULL                  'SHORT'                         5            0  'Not applicable'      'Not applicable'      NULL                  NULL                  'YES'                 NULL                
-1854d1851
+1856d1853
 <   'db_attribute'        'col3'                'max_groupby_order'   'INSTANCE'                      2  NULL                  NULL                  'CHAR'                         30            0  'iso88591'            'iso88591_bin'        NULL                  'abc                           '  'YES'                 NULL                
-1856d1852
+1858d1854
 <   'db_attribute'        'col4'                'max_groupby_order'   'INSTANCE'                      3  NULL                  NULL                  'DATE'                         10            0  'Not applicable'      'Not applicable'      NULL                  'SYS_DATE'            'YES'                 NULL                
-1858d1853
+1860d1855
 <   'db_attribute'        'col5'                'max_groupby_order'   'INSTANCE'                      4  NULL                  NULL                  'CLOB'                          0            0  'Not applicable'      'Not applicable'      NULL                  NULL                  'YES'                 NULL                
-1860d1854
+1862d1856
 <   'db_attribute'        'col6'                'max_groupby_order'   'INSTANCE'                      5  NULL                  NULL                  'MONETARY'                     15            0  'Not applicable'      'Not applicable'      NULL                  NULL                  'YES'                 NULL                
-2106c2100
-< 328 rows selected.Committed.
+2109c2103
+< 329 rows selected.Committed.
 ---
-> 322 rows selected.Committed.
-2186d2179
-<   'db_class'            'max_groupby_order'   'DBA'                 'CLASS'               'NO'                  'NO'                  'NO'                  'iso88591_bin'        NULL                
-2189c2182
-< 46 rows selected.Committed.
+> 323 rows selected.Committed.
+2190d2183
+<   'db_class'            'max_groupby_order'   'DBA'                 'CLASS'               'NO'                  'NO'                  'YES'                 'iso88591_bin'        NULL                
+2193c2186
+< 47 rows selected.Committed.
```

Comment:
This count is 'db_class' table's row count.
The reason for the increase in value is because the dual table was added.